Orange/widgets/visualize/owdistributions.py

```diff
@@ -577,18 +559,17 @@ def onDeleteWidget(self):
         super().onDeleteWidget()
 
     def get_widget_name_extension(self):
-        if self.variable_idx >= 0:
-            return self.varmodel[self.variable_idx]
+        if self.var is not None:
```

Comment:
If this condition is `False`, the method returns `None`, which incidentally equals `self.var`.

Why is the condition necessary, then?